syntaxes/samarium.tmLanguage.json

```diff
@@ -120,7 +120,7 @@
                 },
                 {
                     "name": "keyword.control.samarium",
-                    "match": "(\\*|->|(?<!<)<-|\\.{2,}|@|\\?|!|\\?~>|,,|#|,\\.,|'|\\$|%|<=)"
+                    "match": "(\\*|->|(?<!<)<-|\\.{2,}|@|\\?|!|\\?~>|,,|#|,\\.,|'|\\$|%|<=|~'*)"
```

Comment:
Keep in mind that `*` is a quantifier, so it needs to be escaped
```suggestion
                    "match": "(\\*|->|(?<!<)<-|\\.{2,}|@|\\?|!|\\?~>|,,|#|,\\.,|'|\\$|%|<=|~'\\*)"
```

Reply:
thanks, I have learned so much from these two PR's .

Reply:
Glad to hear that!